src/core.jl

```diff
@@ -59,6 +75,7 @@ parameterlength(a::AbstractArray) = length(a)
 Return the total number of states of the layer `l`.
 """
 statelength(l::AbstractExplicitLayer) = statelength(initialstates(Random.default_rng(), l))
+statelength(l::AbstractExplicitLayer{false, hasparams}) where {hasparams} = 0
```

Comment:
Ordering

Reply:
Fixed in [37f4851](https://github.com/avik-pal/Lux.jl/pull/84/commits/37f48519a731897b9de5771ccb714ff0eea2e013)